muxcore/owner/owner.go

```
@@ -960,6 +1029,67 @@ func (o *Owner) handleDownstreamMessage(s *Session, msg *jsonrpc.Message) error
+		return fmt.Errorf("unexpected message type from downstream: %s", msg.Type)
+	}
+}
+
+func (o *Owner) forwardModernCancelledNotification(s *Session, msg *jsonrpc.Message) error {
+	var envelope struct {
+		Params struct {
+			RequestID json.RawMessage `json:"requestId"`
+		} `json:"params"`
+	}
+	if err := json.Unmarshal(msg.Raw, &envelope); err != nil || envelope.Params.RequestID == nil {
+		return nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 1069) but it returns nil

(nilerr)